tslint.json

```diff
@@ -97,7 +97,6 @@
                 "variable-declaration": "nospace"
             }
         ],
-        "typeof-compare": true,
```

Comment:
Any reason in particular why we don't want to "Makes sure result of typeof is compared to correct string values?"

Reply:
I was seeing a deprecation notice in the console when I was linting, saying that the TypeScript compiler checks this now, so the linting rule is redundant.